### DISCUSSION TITLE
Is LiteFS Cloud used?

### DISCUSSION THREAD

**Discussion by MoSattler** [epicweb-dev/epic-stack#401] [Q&A] [answered]
Hi!

Apologies if this question seems basic or if I've overlooked something obvious, but I noticed that lifefs is in use. Does this include [LifeFS Cloud](https://fly.io/blog/litefs-cloud/)?

**Comment by kentcdodds** [comment, marked as answer]
Yes, you can definitely enable LiteFS Cloud once you've got your app deployed. I do this for my own app and it's great :)
